src/ForceValue.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the zenstruck/foundry package.
+ *
+ * (c) Kevin Bond <kevinbond@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zenstruck\Foundry;
+
+/**
+ * @author NIcolas PHILIPPE <nikophil@gmail.com>
+ *
+ * @internal
```

Comment:
I think it is best to set it `@internal`, so that, the only possibility yo do this would be to use the helper